File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2690.py
n = int(input())

for c in range(n):
    senha = input()
    it = 0
    for i in range(0, len(senha)):
        if(senha[i] != ' '):
            if(senha[i] == 'G' or senha[i] == 'Q' or senha[i] == 'a' or senha[i] == 'k' or senha[i] == 'u'):
                print("0", end = '')
            elif(senha[i] == 'I' or senha[i] == 'S' or senha[i] == 'b' or senha[i] == 'l' or senha[i] == 'v'):
                print("1", end = '')
            elif(senha[i] == 'E' or senha[i] == 'O' or senha[i] == 'Y' or senha[i] == 'c' or senha[i] == 'm' or senha[i] == 'w'):
                print("2", end = '')
            elif(senha[i] == 'F' or senha[i] == 'P' or senha[i] == 'Z' or senha[i] == 'd' or senha[i] == 'n' or senha[i] == 'x'):
                print("3", end = '')
            elif(senha[i] == 'J' or senha[i] == 'T' or senha[i] == 'e' or senha[i] == 'o' or senha[i] == 'y'):
                print("4", end = '')
            elif(senha[i] == 'D' or senha[i] == 'N' or senha[i] == 'X' or senha[i] == 'f' or senha[i] == 'p' or senha[i] == 'z'):
                print("5", end = '')
            elif(senha[i] == 'A' or senha[i] == 'K' or senha[i] == 'U' or senha[i] == 'g' or senha[i] == 'q'):
                print("6", end = '')
            elif(senha[i] == 'C' or senha[i] == 'M' or senha[i] == 'W' or senha[i] == 'h' or senha[i] == 'r'):
                print("7", end = '')
            elif(senha[i] == 'B' or senha[i] == 'L' or senha[i] == 'V' or senha[i] == 'i' or senha[i] == 's'):
                print("8", end = '')
            elif(senha[i] == 'H' or senha[i] == 'R' or senha[i] == 'j' or senha[i] == 't'):
                print("9", end = '')
            
            it += 1
            
            if(it == 12):
                break
            
    print()
        

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/2690.cpp
#include <stdio.h>
#include <string.h>

using namespace std;

int main(){
    int n, k;
    scanf("%d", &n);

    for(k = 0; k < n; k++){
        char senha[64];
        scanf(" %[^\n]", senha);
        
        int i, it = 0;
        for(i = 0; i < strlen(senha) && it < 12; i++){
            if(senha[i] != ' '){
                if(senha[i] == 'G' || senha[i] == 'Q' || senha[i] == 'a' || senha[i] == 'k' || senha[i] == 'u')
                    printf("0");
                else if(senha[i] == 'I' || senha[i] == 'S' || senha[i] == 'b' || senha[i] == 'l' || senha[i] == 'v')
                    printf("1");
                else if(senha[i] == 'E' || senha[i] == 'O' || senha[i] == 'Y' || senha[i] == 'c' || senha[i] == 'm' || senha[i] == 'w')
                    printf("2");
                else if(senha[i] == 'F' || senha[i] == 'P' || senha[i] == 'Z' || senha[i] == 'd' || senha[i] == 'n' || senha[i] == 'x')
                    printf("3");
                else if(senha[i] == 'J' || senha[i] == 'T' || senha[i] == 'e' || senha[i] == 'o' || senha[i] == 'y')
                    printf("4");
                else if(senha[i] == 'D' || senha[i] == 'N' || senha[i] == 'X' || senha[i] == 'f' || senha[i] == 'p' || senha[i] == 'z')
                    printf("5");
                else if(senha[i] == 'A' || senha[i] == 'K' || senha[i] == 'U' || senha[i] == 'g' || senha[i] == 'q')
                    printf("6");
                else if(senha[i] == 'C' || senha[i] == 'M' || senha[i] == 'W' || senha[i] == 'h' || senha[i] == 'r')
                    printf("7");
                else if(senha[i] == 'B' || senha[i] == 'L' || senha[i] == 'V' || senha[i] == 'i' || senha[i] == 's')
                    printf("8");
                else if(senha[i] == 'H' || senha[i] == 'R' || senha[i] == 'j' || senha[i] == 't')
                    printf("9");
                    
                it++;
            }
        }
        
        printf("\n");
    }
    
}